benchmarks/graphics_pipeline/GraphicsBenchmarkApp.cpp

```diff
@@ -499,6 +499,7 @@ void GraphicsBenchmarkApp::UpdateSkyBoxDescriptors()
 
 void GraphicsBenchmarkApp::UpdateSphereDescriptors()
 {
+    GetDevice()->WaitIdle();
```

Comment:
we probably need this line for other update descriptors functions

Reply:
Done